fire_atlas_editor/lib/widgets/image_selection_container.dart

```diff
@@ -55,8 +55,10 @@ class ImageSelectionContainer extends StatelessWidget {
       Container(
           child: FButton(
               label: 'Select image',
-              onSelect: () {
-                selectFile(onSelectImage);
+              onSelect: () async {
+                final storage = FireAtlasStorage();
+                final imgDataUrl = await storage.selectFile(context);
+                onSelectImage(imgDataUrl);
               }))
```

Comment:
trailing comma

Reply:
This is all messed up on the project, I will do a follow up to configure a better linter in the future when we have our flame-lint